src/components/screens/map-screen/map-screen.tsx

```diff
@@ -1277,6 +1289,28 @@ class MapScreen extends React.Component<Props, State> {
     }
   }
 
+  /**
+   * Clear selected locations, routes
+   */
+  private cancelAll = () => {
+    const { loadingRoute } = this.state;
+    this.exitSelectedFavouriteLocation();
+
+    if (loadingRoute) {
+      return;
+    }
+
+    this.setState({
+      routeAltStrict: undefined,
```

Comment:
```suggestion
    this.setState({
      route: undefined,
      routeAltStrict: undefined,
```
Also the selected route should be cleared